cmd/nodecmd/create_aws.go

```diff
@@ -126,15 +130,15 @@ func getAWSCloudConfig(region string, authorizeAccess bool) (*ec2.EC2, string, s
 			}
 		}
 	}
-	sess, err := getAWSCloudCredentials(region, constants.CreateAWSNode, authorizeAccess)
+	sess, err := getAWSCloudCredentials(awsProfile, region, constants.CreateAWSNode, authorizeAccess)
```

Comment:
can the parent of this also receive awsProfile as param? just try to minimize global var usage for flags

Reply:
done